packages/cli/src/services/user.service.ts

```diff
@@ -249,6 +251,18 @@ export class UserService {
 							const newUser = transactionManager.create(User, { email, role });
 							const savedUser = await transactionManager.save<User>(newUser);
 							createdUsers.set(email, savedUser.id);
+							const savedProject = await transactionManager.save<Project>(
```

Comment:
It looks like we are doing a couple of things at user creation, that need to always happen. Maybe we should move this additional transaction inside `UserRepository.create`, so that the projects are also automatically created during integration tests.

Reply:
Do you have a convention for having repositories depend on other repositories? 

For context: In my old place that lead to unsolvable dependency cycles.